gulpfile.js

```diff
@@ -65,6 +65,22 @@ function test() {
 gulp.task('build-test', ['build'], test);
 gulp.task('test', test);
 
+gulp.task('checkImports', function (cb) {
+    var checkProcess = child_process.fork(path.join(__dirname, "tools", "checkCasing.js"),
+        {
+            cwd: path.resolve(__dirname, "src"),
+            stdio: "inherit"
+        });
+    checkProcess.on("error", cb);
+    checkProcess.on("exit", function (code) {
+        if (code) {
```

Comment:
You should compare it with 0. code can also be null indicating that the process didn't exit properly.